search.py

```diff
@@ -326,12 +329,19 @@ def get_boolean_query_scores(field):
         else:
             scores[posting.doc_id] += score
 
+    # Add to the ones judged relevant by humans
```

Comment:
Ok got it!!

Reply:
Wait I thought those judged relevant by humans are meant for query refinement through rocchio? Instead of Query Expansion?

Unless Im seeing wrongly tbh :/

Reply:
Yea. query expansion should only use your thesarus. I'm only using this to give boolean a semblance of ranking